decidim-core/app/packs/src/decidim/dropdown_scrollable.js

```diff
@@ -0,0 +1,10 @@
+$(() => {
```

Comment:
This is not the way we're proposing to work with javascripts. Not only due to you're trusting on JQuery to make it work, but the logic behind is not being initializated in the index.js file, within the initializer function.

In the following lines, you'll notice how we can control when a javascript file is really being used or not:
https://github.com/decidim/decidim/blob/57799762a0a82959e2f8fb2ffb1ea2ca63c68149/decidim-core/app/packs/src/decidim/redesigned_index.js#L157-L177

cc/ @decidim/maintainers